Tools/scripts/build_ci.sh

```diff
@@ -37,6 +38,12 @@ function get_time {
 }
 
 echo "Targets: $CI_BUILD_TARGET"
+echo "Compiler: $c_compiler"
+
+if [ $c_compiler == "clang" ]; then
+    export AUTOTEST_ARGS="--use-clang"
```

Comment:
https://github.com/ArduPilot/ardupilot/pull/9274#discussion_r215461179 still applies. Also, given it's a local one, I'd prefer to have it in lowercase so it's consistent with the others.

Reply:
Done.